packages/libmypaint-gegl.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Libmypaint < Package
```

Comment:
The class name should match the filename.  Rename `libmypaint-gegl.rb` to `libmypaint.rb`?

Reply:
I had originally called it `libmypaint.rb`, but remembered it was actually called `libmypaint-gegl`.
Should I revert to `libmypaint.rb`, or change the class to `Libmypaint_gegl`?

Reply:
Either way is fine as long as the class matches the filename, otherwise, crew will get confused.

Reply:
I'll just omit the `gegl`, keep it shorter.